pkg/database/migrations.go

```diff
@@ -52,12 +52,12 @@ func (db *Database) getMigrations(ctx context.Context) *gormigrate.Gormigrate {
 			Migrate: func(tx *gorm.DB) error {
 				// This is "out of order" as it were, but is needed to bootstrap fresh systems.
 				// Also in migration 8
-				logger.Debugw("db migrations: creating realms table")
+				logger.Debugw("creating realms table")
```

Comment:
How about the logger in this function, does it need the same change?

Reply:
no, debugw is `(args, ...vars)`. It requires vars be an even number (and 0 is even)